lgtm.yml

```diff
@@ -0,0 +1,7 @@
+extraction:
+  go:
+    before_index:
+    - "sed -i 's=$(TOP_DIR)/sonic-mgmt-common=/opt/src/sonic-mgmt-common=g' Makefile"

```

Comment:
Instead of modify Makefile directly, is it possible to override the `MGMT_COMMON_DIR` variable?
I see "LGTM_SRC is /opt/src" in lgtm log.
LGTM support environment variable https://lgtm.com/help/lgtm/customizing-code-extraction

Reply:
After some research and trial/error, changes to Makefile are not needed to have the build succeed with Go language